### DISCUSSION TITLE
10/24 신명희 블로그 생성

### DISCUSSION THREAD

**Discussion by mh0223** [Orm-camp/blog-feedback#21] [동글동글 3기 피드백]
<a href="https://velog.io/@mh0223">https://velog.io/@mh0223<a>

저는 velog로 블로그를 생성했습니다!

초반에는 공부한 내용 정리, 과제 해결 과정 등으로 정리,  후반에는 three.js 위주로 글을 작성해보고 싶습니다,,

적어도 매주 한 번은 작성하도록 해보겠습니다!

**Comment by oratio100** [comment]
블로그 링크가 잘못 연결되어있는 것 같아요~

**Reply by mh0223** [reply]
링크 수정했는데도 안들어가지네요,, 더 찾아보고 수정해볼게요! 알려주셔서 감사합니다!!
